content/docs/command-reference/metrics/show.md

```diff
@@ -42,9 +42,9 @@ compares them with a previous version.
   the workspace. Note that both options can be combined, for example using the
   `-aT` flag.
 
-- `--all-commits` - the same as `-a` or `-T` above, but applies to _all_ Git  
-  commits as well as the workspace. Useful for printing metric file contents for
-  the entire existing commit history of the project.
+- `--all-commits` - same as `-a` or `-T` above, but applies to _all_ Git commits
+  as well as the workspace. Useful for downloading all the data used in the
```

Comment:
why downloading?

Reply:
Bad copy paste probably... Thanks. Fixed in 857919e6.